src/core/server/core_app/bundle_routes/file_hash_cache.ts

```diff
@@ -8,8 +8,22 @@
 
 import LruCache from 'lru-cache';
 
-export class FileHashCache {
-  private lru = new LruCache<string, Promise<string>>(100);
+/** @internal */
+export interface IFileHashCache {
+  get(key: string): Promise<string> | undefined;
+
+  set(key: string, value: Promise<string>): void;
+
+  del(key: string): void;
+}
+
+/** @internal */
+export class FileHashCache implements IFileHashCache {
+  private lru: LruCache<string, Promise<string>>;
+
+  constructor(maxSize: number = 250) {
+    this.lru = new LruCache(maxSize);
+  }
```

Comment:
I kept it for now, but not sure using a `lru` cache still makes sense now, as the number of entries will be constant in production, and we probably don't care much about the cache size increasing in development mode. 

Risk in production, if the cache maxSize is inferior to the number of assets we need to serve on initial load, is that we'll be recomputing hashes for some files every time a user loads Kibana. I increased it from `100` to `250`, but we may just want to use a plain map instead of a lru.

wdyt @spalger?

Reply:
I think the best option would be to compute these values at build time and just load them from a file rather than compute them at run time. The only issue I see with using a `Map` is that the files change in development and the cache keys include the mtime of the file, so the `Map` would just keep growing unless you changed a file server side and the server was restarted. Because of this I'm hesitant to use a `Map` but I think we could probably crank up that maxSize to 1k.

Reply:
Ah, we actually don't even calculate the file hashes in the distributable because the files use unique paths (they have the buildId in the route and are cached client side for 1 year) so the LRU is only used in dev and I think we should keep it.

Reply:
Ok, fine with me